products/models.py

```diff
@@ -26,18 +26,20 @@ class Meta:
 
 class ProductGroup(TimeStamp):
     name            = models.CharField(max_length=50)
+    company         = models.CharField(max_length=50)
     displayed_price = models.DecimalField(max_digits=10, decimal_places=3)
     discount_price  = models.DecimalField(max_digits=5, decimal_places=3)
     sub_category    = models.ForeignKey('SubCategory', on_delete=models.CASCADE)
+    delivery        = models.ForeignKey("Delivery", on_delete=models.CASCADE)
 
     class Meta:
         db_table = 'product_groups'
 
 class Product(TimeStamp):
-    company          = models.CharField(max_length=50)
     name             = models.CharField(max_length=50)
     price            = models.DecimalField(max_digits=10, decimal_places=3)
     product_group    = models.ForeignKey('ProductGroup', on_delete=models.CASCADE)
+    color            = models.ManyToManyField("Color", related_name='products', through="ProductColor")
```

Comment:
나머지는 전부 좋습니다!
마이너한 피드백 하나 드리자면 속성을 ManyToManyField로 선언하실 때에는 복수로 만들어 주셔야 합니다!
예를 들면,
```python
product.color.all() (X)
product.colors.all() (O)

```

```suggestion
    colors            = models.ManyToManyField("Color", related_name='products', through="ProductColor")
```

Reply:
반영하였습니다. 감사합니다!